Sources/LLBuildManifest/ManifestWriter.swift

```diff
@@ -134,6 +134,16 @@ public final class ManifestToolStream {
             stream <<< "    \(key): " <<< Format.asJSON(newValue) <<< "\n"
         }
     }
+
+    public subscript(key: String) -> [String: String] {
+        get { fatalError() }
```

Comment:
would it be better to use a func instead of subscript so we don't have this potential runtime issue?

Reply:
I agree in general but figured it was better to stay with the existing approach for this PR, i.e. the cognitive cost of something completely different for this one case would be more than the benefit.  Seems to me that we should redo all of these at some point.